pkg/antctl/command_definition.go

```diff
@@ -480,6 +494,88 @@ func (cd *commandDefinition) tableOutputForGetCommands(obj interface{}, writer i
 	return nil
 }
 
+/*
+tableOutputForQueryCommands implements printing sub tables (list of tables) for each response, utilizing constructTable
+with multiplicity. tableOutputForGetCommands is the only other user of constructTable, though only one table is printed for
+each response.
+
+*Note this method can be generalized and refactored for any future work which prints sub tables (list of tables).
+*/
+func (cd *commandDefinition) tableOutputForQueryCommands(obj interface{}, writer io.Writer) error {
+	responses := obj.([]*queryendpoint.Response)
```

Comment:
So even if the function name lets one think that this would apply to all "antctl query" commands, it is actually specific to "antctl query endpoint"? That's not a problem for me, but we should rename the function at least.